samples/scaleout/Version_5/Worker2/app.config

```diff
@@ -6,7 +6,7 @@
     <section name="UnicastBusConfig" 
              type="NServiceBus.Config.UnicastBusConfig, NServiceBus.Core" />
   </configSections>
-  <!--startcode WorkerNameToUseWhileTestingConfig-->
```

Comment:
u cant mix language types for snippets. so having one for `WorkerNameToUseWhileTesting` that is a .config will not work with one that is .cs